life_server/Functions/MySQL/fn_bool.sqf

```diff
@@ -1,26 +1,20 @@
 /*
-    File: fn_bool.sqf
-    Author: TAW_Tonic
-
+    File:	fn_bool.sqf
+    Date:   2017-03-22 17:46:27
+    Author: Patrick "Lucian" Schmidt
+    
     Description:
-    Handles bool conversion for MySQL since MySQL doesn't support 'true' or 'false'
-    instead MySQL uses Tinyint for BOOLEAN (0 = false, 1 = true)
+    Converts Bool to Number and Number to bool
```

Comment:
Could you leave the original header in, and place your attribution as an addition :)